README.md

```diff
@@ -1,27 +1,119 @@
 
-### Autogenerated Subtitles
+<a id="mission-of-the-cwl-project"></a>
+### Mission of the CWL Project
+_How we plan to achieve our vision_
 
-* We can optionally downloads YouTube's auto-generated subtitles via https://downsub.com/?url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3D86eY8xs-Vo8
+The CWL project supports open consensus-based standards for command line data analysis
+workflows and tools.
 
+Specifically, we support the
+* pre-standards process by providing a neutral place of convening to discuss, propose and
+test ideas about command-line tool based workflow standards and related topics
+* standardization process by stewarding the development and delivery of standards in
+accordance with the [Open Stand principles](https://open-stand.org/about-us/principles/).  
+* post-standards life cycle by (1) promoting the released standards, (2) developing and maintaining
+related training and tools, and by (3) tracking deficits and other post-standardization feedback.
+<a id="editing-the-site---general-flow"></a>
+
+<a id="editing-the-site---general-flow"></a>
+## Editing the Site - General Flow
+
+1. On `main`, run a `git fetch --all` and pull in any changes.
+2. Switch to `add-jekyll-site` and then merge the changes via `git merge main`
+3. Make sure you're making your changes on the `add-jekyll-site` (or your own branch, which you can merge to `add-jekyll-site`).
+4. Push to `add-jekyll-site` and then check that everything looks good. 
+5. If all is well, head over to the [main repo](https://github.com/common-workflow-language/cwl-website) and create a Pull Request.
+
+<a id="config-files--local-development"></a>
+### Config Files & Local Development
+
+The site uses `_config.yml` for the production site, and `_config_staging.yml` for the staging site (which inherits, then overrides `_config.yml`). For ease of local development, you may wish to use a 3rd config file, e.g. `_config_local.yml`. If so, please don't add it to the repo.
+
+My `_config_local.yml` currently looks like this:
+
+```yaml
+title: CWL Local Site
+url: "cwl.test"
+baseurl: ""
+port: 80
+```
+
+The command for this is: `bundle exec jekyll serve --config "_config.yml,_config_local.yml"`. (Note the lack of spaces between the two config files).
+
+One quirk of Jekyll local development, is that it will set the hostname to `localhost`, ignoring the `url` variable specified in the config file. To override this, you can add the `--host` flag, e.g. `bundle exec jekyll serve --host cwl.test --config "_config.yml,_config_local.yml"`
+
+
+<a id="directory-structure"></a>
+## Directory Structure
+
+As of March 11 2021, the directory structure looks as follows
+
+```sh
+├── _data
+├── _includes
```

Comment:
None of these directories are in the root of this repository, they are in the `content` directory...

Reply:
Updated this. Thx